presto-hive/src/main/java/com/facebook/presto/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -84,6 +84,8 @@
 import static com.facebook.presto.hive.metastore.PrincipalType.USER;
```

Comment:
I just realized that what i'm doing here doesn't make much sense. I should handle a proper schema migration as we do in PageSource and PageSink. Please ignore this commit for now. I'm going to implement proper schema migration as a separate PR.